lighthouse/src/main.rs

```diff
@@ -50,6 +52,18 @@ fn bls_library_name() -> &'static str {
     }
 }
 
+#[inline(always)]
+#[allow(unreachable_code)]
+fn bls_hardware_acceleration() -> bool {
+    #[cfg(feature = "portable")]
+    return false;
+
+    #[cfg(target_arch = "x86_64")]
+    return std::is_x86_feature_detected!("adx");
```

Comment:
This is good for x86_64, but for ARM i.e. aarch64 we need to use [`is_aarch64_feature_detected!`](https://doc.rust-lang.org/std/arch/macro.is_aarch64_feature_detected.html).

Based on a quick read of some BLST issues, I think the feature we want on aarch64 is `neon`.

https://github.com/supranational/blst/issues/213